### DISCUSSION TITLE
Bienvenid@s a las discusiones sobre la Intro a Python!

### DISCUSSION THREAD

**Discussion by FernandaOchoa** [LaunchX-InnovaccionVirtual/CursoIntroPython#21] [Announcements]
## 👋 Hola  @LaunchX-InnovaccionVirtual/mission-innova001 !
  Hace unos días tuvimos un  live en  [el instagram de Innovacción Virtual](https://www.instagram.com/innovaccionvirtual/) resolviendo las dudas más comunes, sobre el ciclo, las cuentas, las contraseñas, los correos y sobre las katas. Es importante nos sigan ahí ya que es dónde normalmente les publicamos información súper rápido.

✨Si son de la idea de no tener instagram, les recomiendo ser más abiertos por que también hacemos tiktoks. ✨

[Aquí puedes ver el Live Instagram](https://www.instagram.com/tv/CZqUtqhAb5S/?utm_medium=copy_link)

Recuerda que tienes disponible una guía con toda la información necesaria para esta semana: 

[GUÍA SEMANA 1](https://fernandaochoa.github.io/innovatemplate/project/onboarding)

---
### Sesiones en vivo:

También les dejo las sesiones que hemos tenido hasta el momento, con la finalidad de que no sea tan caótico la búsqueda de información.

Debes acceder con tu cuenta de innovacción, si no, no podrás ver los lives y no salen a youtube (Sólo el 1ero). 

* [Sesión del viernes 4 de Febrero](https://www.youtube.com/watch?v=Fa2zMaxf2HA)
* [Sesión del lunes 7 de Febrero](https://teams.microsoft.com/l/meetup-join/19%3ameeting_YzAyNzI0YmQtNzIyOS00OWRjLWI4YWMtYzc2MDVhZGFhNGM0%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%227b10324a-57d4-43ce-95fe-68f52b498664%22%2c%22IsBroadcastMeeting%22%3atrue%7d)
* [Sesión del miércoles 9 de Febrero](https://teams.microsoft.com/l/meetup-join/19%3ameeting_MTEzODNjYmYtYmFiYi00NzM3LWJmZDYtYzY3MzUxNmQyMWRh%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%227b10324a-57d4-43ce-95fe-68f52b498664%22%2c%22IsBroadcastMeeting%22%3atrue%7d) : Aquí vamos desde cero con un ejercicio rápido de python.
* [Sesión del viernes 11 de Febrero *Próxima a realizarse](https://teams.microsoft.com/l/meetup-join/19%3ameeting_Yjg2N2NlOWYtNTgyNi00Y2JlLWJmOWQtYjAyZDYwYjViODli%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%227b10324a-57d4-43ce-95fe-68f52b498664%22%2c%22IsBroadcastMeeting%22%3atrue%7d) : Aquí les explico cómo será la dinámica de entrega de las katas y su formato. **NO ANTES**

No se preocupen por hacer pull requests, ni forks en los repositorios de la organización. Lo importante es que practiquen mucho con su cuenta en su perfil. 🎬. Veo muchos forks y muy poquitas estrellitas 💔

### LatinXperts

Cómo parte de su formación, tenemos un extra, esto es la invitación al show LatinXperts.
Son todos los martes a las 7:00pm por el Twitch de GitHub Education: [Clic aquí](https://twitch.tv/githubeducation) en el que compartimos escenario con diferentes personalidades de origen latinx de la industria tecnológica.

--- 
### ¿Cómo externo mis dudas?

A partir de ahora hemos habilitado las discusiones especializadas, puedes consultarlas [Aquí](https://github.com/LaunchX-InnovaccionVirtual/CursoIntroPython/discussions)

Para preguntar algo sigue estos pasos: 

Selecciona crear nueva discusión o 'new discussion': 

![image](https://user-images.githubusercontent.com/9124597/153073154-47e3049e-ae78-4495-ab08-f6cdb61d5371.png)

Después en categoría selecciona la opción 'QnA' o bien 'Preguntas y Respuestas':
![image](https://user-images.githubusercontent.com/9124597/153073316-5588b09f-3738-43c7-9ec6-dc90e891dee7.png)

Con la finalidad de evitar las mismas preguntas, les recomiendo altamente revisar las preguntas ya solucionadas.

---
### Las notificaciones

**Así se desactivan las notificaciones**

![image](https://user-images.githubusercontent.com/9124597/152722707-e50cb83c-69aa-4dbb-9ef7-827d2ece260e.png)

**Comment by MauricioNovelo** [comment]
Que paso con la antigua discusión?
![image](https://user-images.githubusercontent.com/83683322/153082134-119d25cd-2d28-4365-8b19-2e2406749ec1.png)
Se infringió alguna norma?
Tengo esa duda

**Reply by FernandaOchoa** [reply]
Es mucho más organizado trabajar por medio del discussion de un repo. 

Hay categorías, se pueden votar las respuestas y hay un mejor control sobre las notificaciones. 

En innovacción nos encanta iterar constantemente en favor de mejorar la experiencia, este cambio sin duda les facilitará la comunicación.

⚡ Saludos ⚡
